.github/workflows/build_docs.yml

```
@@ -12,20 +12,50 @@ on:
       - 'docs/**'
       - '.github/workflows/build_docs.yml'
   workflow_dispatch:
+  # Build docs if Animate, Goalie, Movement, or UM2N have been updated in the last 24h
+  schedule:
+    - cron: '0 0 * * *'
 
 concurrency:
   group: 'build_docs-${{ github.ref }}'
   cancel-in-progress: true
 
 jobs:
+  # Check if there have been commits on the main branch of other repos in the last 24h
+  check_updates:
+    name: Check for updates in Animate, Goalie, Movement, and UM2N
+    runs-on: ubuntu-latest
+    outputs:
+      updates_found: ${{ steps.check_repos.outputs.updates_found }}
+    steps:
+      - name: Set up environment
+        run: |
+          echo "REPOS=https://api.github.com/repos/mesh-adaptation/animate https://api.github.com/repos/mesh-adaptation/goalie https://api.github.com/repos/mesh-adaptation/movement https://api.github.com/repos/mesh-adaptation/UM2N" >> $GITHUB_ENV
```

Suggest
```suggestion
          WEBPATH="https://api.github.com/repos/mesh-adaptation"
          echo "REPOS=${WEBPATH}/animate ${WEBPATH}/goalie ${WEBPATH}/movement ${WEBPATH}/UM2N" >> $GITHUB_ENV
```
for conciseness

---

Nice! Done in a7b0453b6d40df17fe41ea09d4f7489991fe133a but I named the url `APIURL` instead of `WEBPATH`